client/error-boundary.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from 'react'
+import {polyfill} from 'react-lifecycles-compat'
+
+type ComponentDidCatchInfo = {
+  componentStack: string
+}
+
+export type Info = null | ComponentDidCatchInfo
+
+export type RuntimeError = {
+  ...Error,
+  module: ?{
```

Comment:
I highly recommend using `{| ... |}` over `{ ... }` when possible. I'm not sure if possible here?